lib/class-wp-rest-block-renderer-controller.php

```diff
@@ -37,6 +37,19 @@ public function register_routes() {
 				continue;
 			}
 
+			$layout = array(
+				'layout' => array(
+					'type' => 'string',
+				),
+			);
+
+			// Add layout to attributes since this is used in case of columns block.
+			if ( is_array( $block_type->attributes ) ) {
+				$attributes = array_merge( $block_type->attributes, $layout );
+			} else {
+				$attributes = $layout;
+			}
```

Comment:
Hmm, merging `layout` into the attributes in the controller doesn't feel right to me since it means that the controller is no longer a light layer on top of the `WP_Block_Type_Registry` API.

Maybe we could do this in a getter on `WP_Block_Type`? e.g.

```php
class WP_Block_Type {
	...
	public function get_attributes() {
		if ( is_ array( $this->attributes ) ) {
			return array_merge( $this->attributes, array(
				'layout' => array(
					'type' => 'string',
				),
			) );
		}
		return $this->attributes;
	}
}
```